mi-db/src/influence.rs

```diff
@@ -8,6 +9,15 @@ use utoipa::ToSchema;
 
 use crate::{PG_FOREIGN_KEY_VIOLATION, PG_UNIQUE_KEY_VIOLATION};
 
+#[derive(Debug, FromRow, Clone, Serialize, Deserialize, ToSchema)]
+pub struct InfluenceResponse {
+    #[sqlx(flatten)]
+    #[serde(flatten)]
+    influence: Influence,
+    created_at: chrono::DateTime<Utc>,
+    modified_at: chrono::DateTime<Utc>,
+}
```

Comment:
Problem with this is that, the same struct is used to insert influence data to the db. I tried to wrap the dates in `Option` but since these columns can't be `NULL` in db, the macro rejects it. We have to change time columns to accept null if we want to go that way. If we do it this way, then we can also resolve other requests and change them to macros.